### DISCUSSION TITLE
Move metadata docs into its own part

### DISCUSSION THREAD

**Discussion by LaSylv** [codecentric/spring-boot-admin#3755] [Ideas]
Currently, it's hard to find all the things metadatas offers in the doc, some being in the Spring Boot Admin Client Doc (user.name/user.password), others in their own part (tags) and other implicit through the Spring Boot Maven plugin

As registration metadatas are shared accross all apps, there is nothing specific about any kindof registration, and it can also be used by third party clients like https://github.com/SolarEdgeTech/pyctuator/blob/master/examples/Advanced/README.md

**Comment by joshiste** [comment]
Sounds reasonable. Currently I don't have a good idea how to restructure the docs so that there are not too many duplications and having everything in a place where the reader would expect it. Do you have any suggestions?

**Comment by LaSylv** [comment]
Maybe a specific part in client application, with sub part listing all posibilities. Then for each possibility a note to say from where it can come from ?

Seems hard to find a good way to do it indeed

**Comment by SteKoe** [comment]
Metadata is now documented at https://docs.spring-boot-admin.com/3.5.5/docs/client/configuration
